src/Core/Utilities/StringExtensions.cs

```diff
@@ -30,5 +29,15 @@ public static string RemoveDiacritics(this string text)
                 .ToString()
                 .Normalize(NormalizationForm.FormC);
         }
+
+        public static string FallbackOnNullOrWhiteSpace(this string s, string fallback)
+        {
+            if (string.IsNullOrWhiteSpace(s))
+            {
+                return fallback;
+            }
+
+            return s;
+        }
```

Comment:
⛏️  Can be rewritten if you prefer: 
`public static string FallbackOnNullOrWhiteSpace(this string s, string fallback) => string.IsNullOrWhiteSpace(s) ? fallback : s;`